controllers/builder/openshiftbuilder.go

```diff
@@ -196,6 +196,15 @@ func (o *openshiftBuilderManager) addExternalResources(config *buildv1.BuildConf
 			DestinationDir: workflowRes.WorkflowPath,
 		})
 	}
+	//TODO WM remove this comment
+	//make the workflow properties available to the OpenShift build config.
+	configMapSources = append(configMapSources, buildv1.ConfigMapBuildSource{
```

Comment:
@ricardozanini for the OpenShift build, we do something similar than with the kaniko build task. And we basically add the application.properties to the build.